.github/workflows/build.yml

```
@@ -0,0 +1,122 @@
+        path: playwright-report/
+        retention-days: 30
+
+  deploy:
+    runs-on: [self-hosted, Linux, X64, praevis-web]
+    needs: build
+    if: github.event_name == 'push'
+    
+    steps:
+    - name: Deploy to Development
+      if: github.ref == 'refs/heads/development'
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2088:warning:1:23: Tilde does not expand in quotes. Use $HOME

(shellcheck)